### ISSUE TITLE
Although user has cleared notificaitons, call to displayNotification() displays old notifications

### ISSUE THREAD

**Issue by tad3j** [notifee/react-native-notifee#7]
After playing for the whole day with notiffe, I've noticed at the end of the day `notifee.displayNotification()` started to act a bit funky (hope it's not me sleep deprived :) ). 

Although user has cleared all of the notifications, when new notification arrives and when I call `notifee.displayNotification()` to display new notification arrived, **all of the old notifications will get displayed**. This didn't happen when I started working with  this package, so I assume it could be a bug? It also seems list of "old" notifications being displayed is getting longer and longer.

P.S.: Related to this, I've checked the package and docs to see if I could collapse these notifications with `collapse_key`, but I could find any mentions. I guess collapse keys are not supported at the moment? https://firebase.google.com/docs/cloud-messaging/http-server-ref#downstream-http-messages-plain-text

**Comment by Salakar** [comment]
The whole day on Notifee, nice! Would love your feedback good or bad 😅 if you don't want to give feedback here feel free to sent it to my email (I replied to one of your feedback forms earlier).

In terms of collapse_key, thats an FCM api and there's isn't a "direct" equivalent in Android local notifications APIs, however you can replace a notification with a newer one by using the same `id` for the notifications that you want collapsing (so the id becomes your collapse key effectively), alternatively if you want to visually show all notifications with a collapsible UI then theres `groupId`: https://notifee.app/react-native/reference/notificationandroid#groupid

-----

In terms of display notification keeping history and displaying old notifications; I'm not sure how this is possible from our code, we only accept a single notification as an arg and dont currently store notifications history. Could it be a loop in your event or FCM code perhaps thats triggering more notifications?

**Comment by tad3j** [comment]
Today I had to restart my android emulator and re-build and the issue is gone (not sure why it acted like that yesterday...but issue is not in my code it looks). I will keep an eye on it in case it repeats, so hopefully I am able to reproduce.

Yup. :) Notifee is just awesome, feature rich, with a nice API and great documentation (not only it documents Notifee, also shows how to implement on server side/firebase, so excelent job on that). I've noticed some issues and room for improvement, will send that in email or Discord.

Oh nice, I somehow missed those 2 android notifications properties, will play with them now. Thank you.
